### PR TITLE
Update certificates-scep-configure.md

### PR DESCRIPTION
**Pull request by msbemba**
Mscep.dll is an ISAPI extension that intercepts incoming request and displays the HTTP 403 error if it's installed correctly.

If generic message is displayed then it is a problem

reference - https://learn.microsoft.com/en-us/troubleshoot/mem/intune/troubleshoot-scep-certificate-device-to-ndes#general-ndes-message

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/memdocs/issues/3165

File: memdocs/intune/protect/certificates-scep-configure.md [hunk 1]
```diff
@@ -368,9 +368,7 @@ To configure the NDES service, use an account that is an *Enterprise Administrat
 
 3. Restart the server that hosts the NDES service. Don't use **iisreset**; iireset doesn't complete the required changes.
 
-4. Browse to *http://*Server_FQDN*/certsrv/mscep/mscep.dll*. You should see an NDES page similar to the following image:
-
-   ![Test NDES](./media/certificates-scep-configure/scep-ndes-url.png)
+4. Browse to *http://*Server_FQDN*/certsrv/mscep/mscep.dll*. The result should be: HTTP Error 403.0 – Forbidden. This result indicates the URL is functioning correctly.
```
Comment (JohanFreelancer9) [inline memdocs/intune/protect/certificates-scep-configure.md:371]:
```suggestion
4. Browse to  `http://*Server_FQDN*/certsrv/mscep/mscep.dll`. The result should be **HTTP Error 403.0 – Forbidden**. This result indicates the URL is functioning correctly.
```

File: memdocs/intune/protect/certificates-scep-configure.md [hunk 2]
```diff
@@ -368,9 +368,7 @@ To configure the NDES service, use an account that is an *Enterprise Administrat
...
 
    If the web address returns a **503 Service unavailable**, check the computers event viewer. This error commonly occurs when the application pool is stopped due to a missing [permission for the NDES service account](#accounts).
```
Comment (JohanFreelancer9) [inline memdocs/intune/protect/certificates-scep-configure.md:373]:
```suggestion
   If the web address returns a **503 Service unavailable**, check the computer's event viewer. This error commonly occurs when the application pool is stopped due to a missing [permission for the NDES service account](#accounts).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Brenduns Please review and signoff this PR. Thanks

**Comment by Brenduns** [comment]
#hold-off  Engineering team validation being sought.   The source article has been as it appears today (previously validated by engineering) for about three years, while the Troubleshooting article says something _slightly_ different indicating an issue for that same image which is attributed to a connector install issue - which can be traced back for about two years.    I've got a request to the engineering team to take new look to help us all understand which is correct.  

We also had a change in the Intune connector less than one year ago, but that _shouldn’t_ affect this.  

Once we have validation of the two differing perspectives of that image being bad vs good, we should better understand where a content fix is needed.

**Comment by Brenduns** [comment]
@yogkumgit @msbemba   Looks like we wont' be able to accept this PR. I've had the engineering team take a look, and both the Intune article (this one being modified), and the referenced source article (troubleshootign) have thier information correct:

- The information in each article is correct - and they don't have a conflict.  The original step 4 (In this article the PR addresses) refers to _setting up the NDES infrastructure before the connector is installed_.  Note at the end of that page it has [Download, install, and configure the Certificate Connector for Microsoft Intune](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fmem%2Fintune%2Fprotect%2Fcertificates-scep-configure%23download-install-and-configure-the-certificate-connector-for-microsoft-intune&data=05%7C01%7Cbrenduns%40microsoft.com%7Cdd7c9b13c41a4e784b0308daa6ee263b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638005839024871360%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=AZmuTn18lu8N4IcSi%2BBQgEP%2FqMharlHLbEXs5yzksjM%3D&reserved=0)

- The troubleshooting doc is referring to _after_ you’ve installed the connector software.

As such, there is no reason to edit this article, its information is correct, and I suspect the original PR was based on misunderstanding the differnce between the two articles.

**Comment by Brenduns** [comment]
Closing this PR wihtout further action, as noted back in October.

**Pull request closed without merging**